schemas/webpack.d.ts

```diff
@@ -0,0 +1,360 @@
+        devtoolModuleFilenameTemplate?: string | Function;
+
+        // Specifies the name of each output file on disk. You must **not** specify an absolute path here! The `output.path` option determines the location on disk the files are written to, filename is used solely for naming the individual files.
+        filename?: string;
+        hashDigest?: string;
+        hashDigestLength?: number;
+        hashFunction?: string;
+
+        // The filename of the Hot Update Chunks. They are inside the output.path directory.
+        hotUpdateChunkFilename?: string;
+
+        // The JSONP function used by webpack for async loading of hot update chunks.
+        hotUpdateFunction?: string;
+
+        // The filename of the Hot Update Main File. It is inside the `output.path` directory.
+        hotUpdateMainFilename?: string;
+
+        // The JSONP function used by webpack for async loading of chunks.
+        jsonpFunction?: string;
+
+        // If set, export the bundle as library. `output.library` is the name.
+        library?: string | string[];
+
+        libraryTarget?: OutputLibraryTarget;
+
+        // The output directory as **absolute path** (required).
+        path?: string;
+
+        // Include comments with information about the modules.
+        pathinfo?: boolean;
+
+        // The `publicPath` specifies the public URL address of the output files when referenced in a browser.
+        publicPath?: string;
+
+        // The filename of the SourceMaps for the JavaScript files. They are inside the `output.path` directory.
+        sourceMapFilename?: string;
+
+        // Prefixes every line of the source in the bundle with this string.
+        sourcePrefix?: string;
+
+        // If `output.libraryTarget` is set to umd and `output.library` is set, setting this to true will name the AMD module.
+        umdNamedDefine?: boolean;
+    }
+
+    interface OutputAuxiliaryComment {
+        amd?: string;
+        commonjs?: string;
+        commonjs2?: string;
+        root?: string;
+    }
+
+    interface OutputDevtoolLineToLine {
+        exclude?: string;
+        include?: string;
+        test?: string;
+    }
+
+    interface Resolve {
+        alias?: { key: string } | ResolveAlias[];
+        aliasFields?: [string | string[]];
+        cachePredicate?: Function;
+        descriptionFiles?: string[];
+        enforceExtension?: boolean;
+        enforceModuleExtension?: boolean;
+        extensions?: string[];
+        fileSystem?: any; // TODO missing type
+        mainFields?: [string | string[]];
+        mainFiles?: string[];
+        moduleExtensions?: string[];
+        modules?: string[];
+        plugins?: any[];
+        resolver?: any; // TODO missing type
+        symlinks?: boolean;
+        unsafeCache?: boolean | Object;
+    }
+
+    interface ResolveAlias {
+        alias?: string;
+        name?: string;
+        onlyModule?: boolean;
+    }
+
+    type EntryItem = string | string[];
```

Comment:
My fault, pointed to the wrong line.

I mean for `OutputLibraryTarget` , `Stats` and `Target`.